content/docs/use-cases/ci-and-ml.md

```diff
@@ -0,0 +1,171 @@
+# CI for ML
+
+## Motivation
+
+ML engineers and data scientists face many unique challenges. Some of these
+challenges can be solved by adopting first-citizen concepts from traditional
+software engineering and development. Probably the most common adoption is
+versioning control. Some other, unfortunately less common, could be unit testing
+data processing steps of a pipeline. Here is a non exhaustive list of a few
+other unique challenges:
+
+- Automatically re-train models upon changes.
+- Compare (and report) performance of different versions of a model.
+- Test data correctness when new data is introduced.
```

Comment:
> Test data ... data

Minor: sounds a bit redundant. Maybe use term "dataset" somehow in the 2nd instance? No biggie here, can be left as is (feel free to resolve this comment).

Reply:
Changed :)

Reply:
Thanks!